src/core/sessions/session.service.ts

```
@@ -21,6 +21,9 @@ import { UpdateSessionRequest } from '../../common/dto/sessions/request/update.s
 import { CreateSessionResponse } from '../../common/dto/sessions/response/create.session.response';
 import { GetSessionResponse } from '../../common/dto/sessions/response/get.session.response';
 
+import { CreateSessionResponse } from '../../common/dto/sessions/response/create.session.response';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'CreateSessionResponse'. Consider to delete it or rename it.

'CreateSessionResponse' is defined here:


(lint/suspicious/noRedeclare)